resources/META-INF/plugin.xml

```diff
@@ -262,7 +262,6 @@
          | Compiler
          -->
 
-        <applicationService serviceImplementation="com.reason.ide.ORCompilerManager"/>
```

Comment:
I'm not sure, it seems it should be a projectservice like other services, instead of a regular component

Reply:
figured out the issue here. I need to reference the project so this has to be switched to a `projectService`. Reverting and converting this to a `projectService`.